stubs/antlr4-python3-runtime/METADATA.toml

```
@@ -0,0 +1,5 @@
+version = "4.13.*"
+upstream_repository = "https://github.com/antlr/antlr4"
+
+[tool.stubtest]
```

While we obviously prefer stubs to be as complete as possible, it's fine to do this for now if you want stubtest to shut up about things being missing from the stub even though they're present at runtime

```suggestion
[tool.stubtest]
ignore_missing_stub = true
```